Velaptor/Input/Mouse.cs

```diff
@@ -64,8 +65,8 @@ public MouseState GetState()
     /// Updates the state of the mouse.
     /// </summary>
     /// <param name="data">The mouse change data.</param>
-    /// <exception cref="EnumOutOfRangeException{MouseButton}">
-    ///     Thrown if the mouse button enum is out of range.
+    /// <exception cref="InvalidEnumArgumentException">
+    ///     Occurs if the <see cref="MouseStateData.Button"/> is an invalid value.
```

Comment:
Please refactor this XAML doc to show `MouseStateData` as well as `Button`.

```suggestion
    ///     Occurs if the <see cref="MouseStateData"/>.<see cref="MouseStateData.Button"/> is an invalid value.
```